src/Microsoft.ML.Vision/VisionCatalog.cs

```diff
@@ -3,13 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.ML.Data;
-using Microsoft.ML.Dnn;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Transforms;
+using Microsoft.ML.Vision;
+using static Microsoft.ML.TensorFlow.TensorFlowUtils;
 
 namespace Microsoft.ML
 {
-    /// <include file='doc.xml' path='doc/members/member[@name="DnnRetrainTransformer"]/*' />
     public static class DnnCatalog
```

Comment:
Should this be renamed to `VisionCatalog`?

Reply:
oh sorry, my bad, I thought I had renamed it.

Reply:
It shows I have renamed it on my side!

Reply:
It is not showing as renamed in the review. You renamed the file, but not the class.

---
In reply to: [340166729](https://github.com/dotnet/machinelearning/pull/4408#discussion_r340166729) [](ancestors = 340166729)